build.xml

```diff
@@ -130,122 +130,15 @@
   <target name="_package">
     <mkdir dir="${package.output.dir}"/>
     <exec
-      command="${pear.exec} channel-discover pear.marcelog.name"
-      dir="${project.basedir}" passthru="true"
-    />
-    <exec
       command="${genphar.exec} ${genphar.args}"
```

Comment:
Yeah, it can still be generated. Although it wont be "officially" supported/distributed.